wiki/configuration.md

```diff
@@ -13,7 +13,7 @@ to what object is configured using a set of rules.
 
 In this case we use the same instance of RabbitMQ and MongoDB.
 
-  - **rabbitmq.consumerName** property should be different for each rule set. 
+  - **rabbitmq.consumer.name** property should be different for each rule set.
```

Comment:
I'm missing a section explaining the rabbitMQ properties. If we don't explain them all we could at least link to some external documentation. We used to have comments in the application.properties about TLS versions etc, that information should still be stated somewhere I think.

Reply:
I found the docs related to allowed security protocols here: https://docs.oracle.com/javase/7/docs/technotes/guides/security/StandardNames.html#SSLContext
Do you want me to write more about the rabbitmq properties?

Reply:
I wrote briefly about the properties that are not obvious, host, port and such, that are commonly known rabbit mq properties I didn't bother to explain.